src/main/java/org/joda/railmodel/Station.java

```diff
@@ -1,95 +1,11 @@
-/*
- *  Copyright 2015-present Stephen Colebourne
- *
- *  Licensed under the Apache License, Version 2.0 (the "License");
- *  you may not use this file except in compliance with the License.
- *  You may obtain a copy of the License at
- *
- *      http://www.apache.org/licenses/LICENSE-2.0
- *
- *  Unless required by applicable law or agreed to in writing, software
- *  distributed under the License is distributed on an "AS IS" BASIS,
- *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- *  See the License for the specific language governing permissions and
- *  limitations under the License.
- */
 package org.joda.railmodel;
 
 /**
- * Stations.
+ * @author Sleiman
+ *         <p/>
+ *         Date: 12/12/2015.
```

Comment:
This file needs its copyright back. Please add yourself to the contributors section in the pom.xml and remove the author tag lines here. thanks